content/rest/push.textile

```diff
@@ -0,0 +1,56 @@
+---
```

Comment:
THis content is duplicated, should we use a partial to avoid these diverging in future?